specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -97,6 +97,114 @@ export const attachmentsText = createDictionary({
     'uk-ua': 'Показати форму',
     'de-ch': 'Formular anzeigen',
   },
+  multipleFilesSelected: {
+    'en-us': 'Multiple files selected',
+  },
+  fileSize: {
+    'en-us': 'File Size',
+  },
+  deleted: {
+    'en-us': 'Deleted',
+  },
+  noFile: {
+    'en-us': 'No File',
+  },
+  incorrectFormatter: {
+    'en-us': 'Incorrectly Formatted',
+  },
+  alreadyUploaded: {
+    'en-us': 'Already Uploaded',
+  },
+  alreadyDeleted: {
+    'en-us': 'Already Deleted',
+  },
+  skipped: {
+    'en-us': 'Skipped',
+  },
+  cancelled: {
+    'en-us': 'Cancelled',
+  },
+  frontEndInterruption: {
+    'en-us': '{action:string} was in progress when interruption occured',
+  },
+
```

Comment:
note that these new lines (or commens you add in here) won't be preserved when Weblate integration code gets its hands on these localization files (it also reformats the multiple line strings)

just a FYI in case you wanted to use new lines for semantic separation

cc @CarolineDenis @grantfitzsimmons @melton-jason

Reply:
thanks for letting us know!